dense-numericals-src/basic-math/arg-minimum.lisp

```diff
@@ -129,26 +122,35 @@
                                           ((keep-dims null)))
     (signed-byte 64)
   (declare (ignore axis out keep-dims))
-  (pflet ((acc     -1)
+  (pflet ((val (type-max <type>))
           (c-name  (c-name <type> 'nu:arg-minimum))
+          (c-val-name  (c-name <type> 'nu:minimum))
           (c-size  (c-size <type>)))
-    (declare (type (signed-byte 64) acc))
-    (ptr-iterate-but-inner (narray-dimensions x)
-        n
-      ((ptr-x c-size inc-x x))
-      (setq acc
-            (cl:min acc (inline-or-funcall c-name n ptr-x inc-x))))
-    acc))
+    (declare (type <type> val))
+    (cl:let ((acc     -1)
+             (offset   0)
+             (stride (reduce #'* (cdr (narray-dimensions x)))))
```

Comment:
The same changes that were made to `arg-maximum` wrt stride->nlast and two c-calls -> single c-call can be made to `arg-minimum` right?

Also the make-view -> as-displaced-array in arg-minimum.lisp. Even with the as-displaced-array in arg-maximum.lisp, we can call the `nu:asarray` from within the as-displaced-array, right? Something like the following. The make-view -> as-displaced-array is just aesthetics though.

```lisp
(as-displaced-array (array-like)
  (let ((array (nu:asarray array-like)))
    (make-array (array-dimensions array)
                :element-type (array-element-type array)
                :displaced-to array))))
```